app/src/main/java/org/p2p/wallet/history/ui/history/HistoryContract.kt

```diff
@@ -12,13 +12,17 @@ interface HistoryContract {
         fun openTransactionDetailsScreen(transaction: HistoryTransaction)
         fun showRefreshing(isRefreshing: Boolean)
         fun scrollToTop()
-        fun showHistory(blockchainTransactions: List<HistoryTransaction>, sellTransactions: List<SellTransaction>)
+        fun showHistory(
+            blockChainTransactions: List<HistoryTransaction>,
+            sellTransactions: List<SellTransaction>
+        )
     }
 
     interface Presenter : MvpPresenter<View> {
         fun loadHistory()
         fun refreshHistory()
         fun loadNextHistoryPage()
         fun onItemClicked(transaction: HistoryTransaction)
+        fun onResume()
```

Comment:
you can extend your Presenter from DefaultLifecycleObserver
and attach it in fragment using lifecycle.addObserver(presenter) in onViewCreatedMethod()

Reply:
i think it's a small overhead, i need only a method to call when fragment onResumed

Reply:
it provides you onResume onDestroy methods, to your presenter. By the way x D

Reply:
let's add more context to it's name. 
something like `updateHistory`. 
I think `onResume` is bad naming for it